Assets/Scripts/HandController.cs

```diff
@@ -43,7 +44,7 @@ public class HandController : MonoBehaviour {
             DeselectCard (cardController);
         }
 
-        bool canCast = castButton.OnSelectionChanged (selectedCards.Count);
+        bool canCast = combatManager.ParseSpell (selectedCards.Count);
 
         UpdateAllSelectedCards (canCast);
```

Comment:
Since we're only referring to `canCast` as a method argument, maybe we should just replace it with what we're calling in the first place:
```suggestion
        UpdateAllSelectedCards (combatManager.ParseSpell (selectedCards.Count));
```

Reply:
This is not a hill I'm willing to die on though, if you feel like this gets too unreadable I can go either way.

Reply:
Yeah I feel like with condensing boolean expressions we can quickly get into unreadable territory. Especially since `ParseSpell` doesn't intuitively seem to return a boolean.

This could be resolved by naming `ParseSpell` better though.... thoughts?

Reply:
Hmm I'll rename it to `isValidSpell()` during my PR, but for now we can shorthand it like this.